_projects/2024/sunpy/eclipse.md

```diff
@@ -0,0 +1,63 @@
+---
+name: Solar Eclipse analysis with sunpy
+desc: Process and analysis images taken of the total solar eclipse that crosses the US in 2024
+requirements:
+ - Experience with Python
+difficulty: medium to high
+issues:
+ - https://github.com/sunpy/sunpy/issues/7387
+mentors:
+ - nabobalis
+ - ehsteve
+initiatives:
+ - GSOC
+project_size:
+ - 175 h / medium
```

Comment:
This is possibly a large project.

Reply:
I would say pick one for the listing, and you will work with the applicants anyway during the application period, they can put the size in that you see fit then.

Reply:
Sure, I am hoping that some of the other mentors on this project when they review it will help decide if this large or medium so both is not on the proposal.

Reply:
OK. whenever you flip the switch from draft I'll merge.